doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -519,7 +519,7 @@ but moderate. The minimum timeouts should never be below 10 seconds.
 Location to store the resource state in.
 </longdesc>
 <shortdesc lang="en">State file</shortdesc>
-<content type="string" default="/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state" />
+<content type="string" default="/var/run//Dummy-default.state" />
```

Comment:
On 23/02/16 08:24 -0800, Ken Gaillot wrote:
> Explaining in longdesc sounds like a good idea. Personally, if I saw
> ${} in help output, I would just think some code slipped through by
> mistake.

+1

> I don't think any code uses the default from the metadata.

Actually I was really surprised that in Pacemaker (and I suppose also
Heartbeat) style of integration, default values are more or less internal
info to the RAs themselves and they get exposed in metadata as a mere
expression of good manners.

Compare that to the RGManager style, where agents are dumb about their
very own declared defaults as they require those to be fed back by the
cluster resource manager anytime they are not overridden in the config
(apparently, custom values are fed, then; point is: full set of
parameters is always passed in, possibly consisting solely of defaults).

-- 
Jan (Poki)